lib/osml/model_runner/monitoring/mr_monitoring.ts

```diff
@@ -89,255 +92,146 @@ export class MRMonitoring extends Construct {
-
-    // This comes from image_utils.py in the ModelRunner code. We include
-    // image format as a dimension not image type.
-    ["TIFF", "NITF"].forEach((imageFormat) => {
-      this.dashboard.addWidgets(
-        this.generateProcessingWidget(
-          imageFormat,
-          props.account.region,
-          props.mrDataplaneConfig.METRICS_NAMESPACE
-        )
-      );
-    });
-
-    if (this.modelStatsWidget != undefined) {
-      this.dashboard.addWidgets(this.modelStatsWidget);
-    }
-
-    this.dashboard.addWidgets(
-      featureMetricsWidget,
-      ecsClusterUtilizationWidget
+      })
     );
-  }
 
-  /**
-   * Builds a metric widget in Cloudwatch Dashboard
-   * @param metricName
-   * @param label
-   * @param imageFormat
-   * @param metricsNamespace metricsNamespace the name to apply to the metric namespace
-   * @param statistic
-   * @returns: cloudwatch metric
-   */
-  buildProcessingMetric(
-    metricName: string,
-    label: string,
-    imageFormat: string,
-    metricsNamespace: string,
-    statistic?: string
-  ): Metric {
-    return new Metric({
-      namespace: metricsNamespace,
-      metricName,
-      dimensionsMap: {
-        ImageFormat: imageFormat
-      },
-      statistic,
-      label
-    });
-  }
-
-  /**
-   * Generates a widget that contains metrics for processing images, regions, and tiles
-   * @returns: cloudwatch Widget
-   * @param imageFormat
-   * @param region
-   * @param metricsNameSpace
-   */
-  generateProcessingWidget(
-    imageFormat: string,
-    region: string,
-    metricsNameSpace: string
-  ): SingleValueWidget {
-    return new SingleValueWidget({
-      title: `Processing Stats - ${imageFormat}`,
-      region,
-      width: 24,
-      height: 3,
-      sparkline: true,
-      metrics: [
-        this.buildProcessingMetric(
-          "RegionsProcessed",
-          "Regions Processed (Total: ${SUM})",
-          imageFormat,
-          metricsNameSpace,
-          "sum"
-        ),
-        this.buildProcessingMetric(
-          "TilingLatency",
-          "Tiling Latency",
-          imageFormat,
-          metricsNameSpace,
-          "avg"
-        ),
-        this.buildProcessingMetric(
-          "RegionLatency",
-          "Regions Latency",
-          imageFormat,
-          metricsNameSpace,
-          "avg"
-        ),
-        this.buildProcessingMetric(
-          "TilesProcessed",
-          "Tiles Processed (Total: ${SUM})",
-          imageFormat,
-          metricsNameSpace,
-          "sum"
-        ),
-        this.buildProcessingMetric(
-          "ImageProcessingError",
-          "Processing Failures (Total: ${SUM})",
-          imageFormat,
-          metricsNameSpace,
-          "sum"
-        )
-      ]
-    });
+    this.mrDashboard.addWidgets(
+      new LogQueryWidget({
+        title: "Image Status",
+        width: 24,
+        height: 11,
+        region: props.account.region,
+        logGroupNames: ["/aws/OSML/MRService"],
```

Comment:
Just noting it might be nice to abstract this out to a configuration in the future too.